src/zkaf/apps/src/bin/publisher.rs

```diff
@@ -20,36 +24,46 @@ fn main() -> Result<()> {
 
     // // TODO call image generation service here
     // generate the NFT payload
-    let (_request, response) = get_http_payload(proof_params.clone());
-    let (nft_payload, stringified_nft_payload) = generate_tweet_nft_payload(response);
+    let (response, _request) = verify_proof(&proof_params.proof.clone()).unwrap();
```

Comment:
If the `proof_params` derived from `zk_params.json` is fixed, and no longer dynamically generated in `build.rs`, then it should live in the `fixtures` folder.